fake_zocalo/__main__.py

```diff
@@ -72,6 +77,22 @@ def main():
         "recipe-pointer": 1,
     }
 
+    multi_crystal_result = {
```

Comment:
Nit: This is getting a bit messy with these 3 basically the same maybe something like:
```python
def make_result(payload):
    return {
        "environment": {"ID": "6261b482-bef2-49f5-8699-eb274cd3b92e"},
        "payload": payload,
        "recipe": {
            "start": [[1, payload]],
            "1": {
                "service": "Send XRC results to GDA",
                "queue": "xrc.i03",
                "exchange": "results",
                "parameters": {"dcid": "2", "dcgid": "4"},
            },
        },
        "recipe-path": [],
        "recipe-pointer": 1,
    }

results = defaultdict(lambda : make_result([TEST_RESULT_LARGE]), {"NO_DIFF": make_result([]), "MULTI_X": make_result([*TEST_RESULT_LARGE, *TEST_RESULT_SMALL]})

...

result = results[prefix]
```